app/api/ada/adaLocalStorage.js

```diff
@@ -45,8 +45,11 @@ export function saveCryptoAccount(
   _saveInStorage(storageKeys.ACCOUNT_KEY, localAccount);
 }
 
-export function getCurrentCryptoAccount(): CryptoAccount {
+export function getCurrentCryptoAccount(): ?CryptoAccount {
```

Comment:
Although I agree that conceptually we don't always have an account, in practice in Yoroi we always do after account creation so I don't really get why this was necessary. It makes the code a messier, causes merge conflicts with the storage decouple PR and we'll have to redo all of this with the new API anyway

Reply:
Discussed with @SebastienGllmt - there's logical case when `getCurrentCryptoAccount` might be called when account is not initialised yet, and it was happening with the new code when I added new call to this function. I am ok to hacking around without changing these types for now, if it's gonna change anyways